### PR TITLE
Split locations/js/widgets_main based on select2 version

### PR DESCRIPTION
**Pull request by orangejenny**
Similar to https://github.com/dimagi/commcare-hq/pull/21770, split this module into a v3 and a v4 version so I can upgrade pages one at a time.

Product note: this makes trivial changes to the look and feel of the select2 widgets when adding or editing a repeater, see below.

Marking don't merge just until I sanity test on staging.

@pr33thi / @millerdev 

before
screen shot 2018-09-07 at 9 27 11 pm

after
screen shot 2018-09-07 at 9 25 04 pm

### CODE REVIEW

File: corehq/apps/users/templates/users/base_template.html [hunk 1]
```diff
@@ -3,5 +3,5 @@
 {% load hq_shared_tags %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static "locations/js/widgets_main.js" %}"></script>
```
Comment (pr33thi) [inline corehq/apps/users/templates/users/base_template.html]: Looks like a quotation syntax error

File: custom/ewsghana/templates/ewsghana/user_extension.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static "locations/js/widgets_main.js" %}"></script>
```
Comment (pr33thi) [inline custom/ewsghana/templates/ewsghana/user_extension.html]: Quote syntax

File: corehq/apps/users/templates/users/base_template.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/users/templates/users/base_template.html]: I don't understand - what's the error? I thought github diff just doesn't know quite how to highlight these files.

Comment (pr33thi) [reply corehq/apps/users/templates/users/base_template.html]: Oh cool, then you can disregard, I was just trusting github highlighting!

**Review by millerdev — APPROVED** [review]

**Pull request merged**